miniconda/Dockerfile

```diff
@@ -3,8 +3,9 @@ FROM debian:7.4
 MAINTAINER Travis Swicegood
 
 RUN apt-get update && apt-get install -y wget bzip2
-RUN echo 'export PATH=/opt/anaconda/bin:$PATH' > /etc/profile.d/conda.sh
-RUN wget --quiet http://repo.continuum.io/miniconda/Miniconda3-3.6.0-Linux-x86_64.sh && \
-    /bin/bash /Miniconda3-3.6.0-Linux-x86_64.sh -b -p /opt/anaconda && \
-    rm Miniconda3-3.6.0-Linux-x86_64.sh
+RUN echo 'export PATH=/opt/anaconda/bin:$PATH' > /etc/profile.d/conda.sh && \
+    wget --quiet http://repo.continuum.io/miniconda/Miniconda-3.7.3-Linux-x86_64.sh && \
+    /bin/bash /Miniconda-3.7.3-Linux-x86_64.sh -b -p /opt/anaconda && \
```

Comment:
It was something I did on an early version of building this.  The idea is that you can mount an `/opt/anaconda/pkgs` and/or `/opt/anaconda/envs` directory in to an image based off of Miniconda and have everything.  I'd actually like to just change it entirely to be `/opt/conda`, but that's a whole other discussion.